src/pages/about/notices.md

```diff
@@ -71,103 +73,104 @@ The program described in this document also includes the vocabulary sources list
 |SNMI98|Cote, Roger A., Systematized Nomenclature of Human and Veterinary Medicine: SNOMED International, Version 3.5, Schaumberg (IL), College of American Pathologists, American Veterinary Medical Association, 1998|
-|SNOMEDCT_US_2021_03_01 (updated)|SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, March 1, 2021|
-|SNOMEDCT_US_2017_01_31|SNOMED International, SNOMED Clinical Terms US Edition, January 31, 2017, London, United Kingdom, SNOMED International|
-|SNOMEDCT_US_2018_03_01|SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, March 1, 2018|
+|SNOMEDCT_US_2022_03_01 (updated)|SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, March 1, 2022|
+|SNOMEDCT_US_2021_09_01 (updated)|SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, September 1, 2021|
 |SNOMEDCT_US_2020_07_31|SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, July 31, 2020|
-|SNOMEDCT_VET_2020_10_08 (updated) |SNOMED International, US Edition of SNOMED CT, London, United Kingdom, SNOMED International, March 1, 2021|
+|SNOMEDCT_VET_2021_10_01 (updated)|Veterinary Terminology Services Laboratory (VTSL),Virginia Maryland Regional College of Veterinary Medicine, Veterinary Extension to SNOMED CT, Blacksburg, VA, October 2021|
 |SOP9.2 (updated)|National Association of Health Data Organizations (NAHDO),Source of Payment Typology, Version 9.2, National Association of Health Data Organizations (NAHDO), December, 2020|
 |SPN2003|Standard Product Nomenclature (SPN). Rockville, (MD): U.S. Food and Drug Administration, 2003|
 |SRC|UMLS Metathesaurus Source Terminologies. Bethesda, MD: National Library of Medicine|
 |TKMT2011|Traditional Korean Medical Terms, 2011 Korea, KOR|
-|USP_2021_02_15 (updated)|USP Compendial Nomenclature, February, 2021, United States Pharmacopeia, February, 2021|
+|USP_2022_02_15 (updated)|USP Compendial Nomenclature, February, 2022, United States Pharmacopeia, February, 2022|
 |USPMG_2020|USP Medicare Model Guidelines, Version 6, United States Pharmacopeia February 1, 2020|
 |UWDA173|Jose Mejino, M.D., University of Washington Digital Anatomist, (UWDA). Seattle (WA): University of Washington, Version 1.7.3, March, 2003.|
-|VANDF_2021_01_29 (updated)|Veterans Health Administration National Drug File, January 29, 2021 Washington, DC U.S. Department of Veterans Affairs|
+|VANDF_2022_01_31 (updated)|Veterans Health Administration National Drug File, January 31, 2022, Washington, DC U.S. Department of Veterans Affairs|
 
 Deidentified clinical records used in the development of this offering were provided by the i2b2 National Center for Biomedical Computing funded by U54LM008748 and were originally prepared for the Shared Tasks for Challenges in NLP for Clinical Data organized by Dr. Ozlem Uzuner, i2b2 and SUNY
 
```

Comment:
Can you double-check on a few of the mentioned dates starting on line 177...
1). "1995-2019" is mentioned in 2 diff places at line 177.  The current LOINC license "https://loinc.org/kb/license/" says 1995-2022, but these may be referring to different things.  At line 177, it splits it by "LOINC Table and LOINC codes" and by "LOINC Table, LOINC Table Core, LOINC Panels and Forms File...."
2). At the same website, under Notice of Third Party Content..." bullet E, says "1995-2020".  Should our line 179 in this document list the same years?
(The years listed on Line 181 and 183 match Bullets F and G, so those dates look correct).